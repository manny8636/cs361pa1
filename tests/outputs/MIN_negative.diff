--- outputs/MIN_negative.txt	2022-02-15 14:30:52.839050570 -0500
+++ expected/MIN_negative.txt	2021-01-25 15:27:24.000000000 -0500
@@ -0,0 +1,2 @@
+VALUE [integer]: -25
+Success!
