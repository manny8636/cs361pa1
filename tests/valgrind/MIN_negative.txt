==3062702== Memcheck, a memory error detector
==3062702== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3062702== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==3062702== Command: ../json -v ../data/negative.txt
==3062702== 
Success!
==3062702== 
==3062702== HEAP SUMMARY:
==3062702==     in use at exit: 4 bytes in 1 blocks
==3062702==   total heap usage: 4 allocs, 3 frees, 16,860 bytes allocated
==3062702== 
==3062702== LEAK SUMMARY:
==3062702==    definitely lost: 4 bytes in 1 blocks
==3062702==    indirectly lost: 0 bytes in 0 blocks
==3062702==      possibly lost: 0 bytes in 0 blocks
==3062702==    still reachable: 0 bytes in 0 blocks
==3062702==         suppressed: 0 bytes in 0 blocks
==3062702== Rerun with --leak-check=full to see details of leaked memory
==3062702== 
==3062702== For lists of detected and suppressed errors, rerun with: -s
==3062702== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
