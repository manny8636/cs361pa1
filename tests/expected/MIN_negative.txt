VALUE [integer]: -25
Success!
